lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -371,7 +371,7 @@ def add_address(post, options)
 
       def add_statement_address(post, options)
         return unless statement_address = options[:statement_address]
-        return unless [:address1, :city, :zip, :state].all? { |key| statement_address[key].present? } 
+        return unless [:address1, :city, :zip, :state].all? { |key| statement_address[key].present? }
```

Comment:
I was sure Ruby had to have an equivalent of Python's `not any({'address1', 'city', 'zip', 'state'} - statement_address.keys())` but nope, this seems like the Ruby way.